### PR TITLE
Addnotes

### PR DESCRIPTION
**Pull request by Richa-rani02**
Added Below features:
1. create new Notes
2. Pick color of notes
3. create label 
4. choose priority
5. Archive note, move note from archive to trash,restore archive note
6. move note to trash, restore from trash
7. Created backend Api for trash functionality
notesdemo

### CODE REVIEW

File: src/components/TextEditor.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      Color:{tagColor:"#a1a1aa", bgColor:"#f4f4f5"},
+      priority:"",
+      createdTime:date,
+      error:"",
+  }
+  const [notes,setNotes]=useState({...noteInput})
+
+ const AddNotes=(e)=>{
+   e.preventDefault();
+   if(notes.title.length==0 && notes.description.length==0){
```
Comment (JantuDeb) [inline src/components/TextEditor.jsx]: it should be `||`

**Review by JantuDeb — APPROVED** [review]
Overall this looks fine

File: src/components/TextEditor.jsx (hunk 1 above)
Comment (Richa-rani02) [reply src/components/TextEditor.jsx]: Thanks for pointing out fixed it.

**Review by anshikaa12 — APPROVED** [review]
liked the UI
clean code!

**Pull request merged**